### ISSUE TITLE
Improve output readability: warnings vs errors

### ISSUE THREAD

**Issue by etanshaul** [ChrisCarini/intellij-platform-plugin-verifier-action#17]
Hello! we are successfully using this verifier on our plugin's repo. We have it configured currently so that only `compatibility_problems` stop the build. We do unfortunately have a lot of deprecation warnings as well that we intend to fix at some point. However, it is currently quite hard to parse the output of the check to get at the actual blocking issue.

Request:

Format the verification output in such a way that makes it easier to get out the violations of the configured `failure_levels`. I'm leaving it open as to how that would look, but currently we have to scroll through a lot of logs / make extensive use of `ctrl`+`f`.

**Comment by ChrisCarini** [comment]
Hi @etanshaul - this request makes sense at first reading. Do you have any GH workflow execution(s) you are able to share that I could look at to get a better sense of the painpoint here?

**Comment by ChrisCarini** [comment]
@all-contributors please add @etanshaul for ideas
